Multisig/UI/Settings/OwnerKeyManagement/OwnerKeyDetailsViewController/OwnerKeyDetailsViewController.swift

```diff
@@ -172,7 +172,31 @@ class OwnerKeyDetailsViewController: UITableViewController {
     }
 
     @objc private func reloadData() {
-        DispatchQueue.main.async {
+        DispatchQueue.main.async { [unowned self] in
+            self.sections = [
+                (section: .name("OWNER NAME"), items: [Section.Name.name]),
+
+                (section: .keyAddress("OWNER ADDRESS"),
+                 items: [Section.KeyAddress.address]),
+
+                (section: .ownerKeyType("OWNER TYPE"),
+                 items: [Section.OwnerKeyType.type])]
+
+            if self.keyInfo.keyType == .walletConnect {
+                self.sections.append((section: .connected("WC CONNECTION"), items: [Section.Connected.connected]))
+            }
+
+            if [.walletConnect, .ledgerNanoX].contains(keyInfo.keyType) {
+                self.sections.append((section: .pushNotificationConfiguration("PUSH NOTIFICATIONS"),
+                                 items: [Section.PushNotificationConfiguration.enabled]))
+                if self.keyInfo.delegateAddress != nil {
```

Comment:
I think we'll need to re-do this part: we need to ask backend what delegate key exist on each network and display the list either here or in a separate screen... 

This is not what I intended, but it still works I guess for now.

Reply:
[Ticket](https://app.zenhub.com/workspaces/mobile-development-board-60cc78740f488d000e996335/issues/gnosis/safe-ios/1680)